### ISSUE TITLE
Total isk loss is not updated on "calculateIskLoss"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [EVE-SECURE/evedev-kb#107]
```
File: class.kill.php
Function: calculateISKLoss (~Line 928)

kb3_kills.kll_isk_loss is not updated.

Should probably do something like:
if($update)
{
    $qry = DBFactory::getDBQuery();
    $qry->execute("UPDATE kb3_kills SET kll_isk_loss = '$value' WHERE 
kll_id = '".$this->id_."'");
}

/Beanie
```

Original issue reported on code.google.com by `niels%ns...@gtempaccount.com` on 16 Apr 2010 at 9:26
